pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -346,21 +349,29 @@ func (k *Kubernetes) ConfigServicePorts(name string, service kobject.ServiceConf
 		targetPort.IntVal = port.ContainerPort
 		targetPort.StrVal = strconv.Itoa(int(port.ContainerPort))
 
+		// decide the name based on whether we saw this port before
+		name := strconv.Itoa(int(port.HostPort))
+		if _, ok := seenPorts[int(port.HostPort)]; ok {
+			// https://github.com/kubernetes/kubernetes/issues/2995
+			if service.ServiceType == string(api.ServiceTypeLoadBalancer) {
+				panic("Services of type LoadBalancer cannot use both TCP and UDP for same port " + name)
```

Comment:
This function should really be returning an `err`, but as of right now, that's in a future refactor.

Instead of a panic, can you please use `log.Fatalf` ? Would be helpful to output which service we're using too.

```go
log.Fatalf("Service %s of type LoadBalancer cannot use TCP and UDP for the same port", name)
```